apps/hub-app/src/pages/PublicProfilePage.tsx

```diff
@@ -151,12 +169,12 @@ const PublicProfilePage = () => {
             <Profile />
             {daoData.length < 1 ? (
               <DaoCountHeading>
-                <ParMd>No Dao memberships found</ParMd>
+                <ParMd>No DAO memberships found</ParMd>
               </DaoCountHeading>
             ) : (
               <DaoData>
                 <DaoCountHeading>
-                  {daoData.length} Total Dao {daoData.length === 1 ? '' : 's'}
+                  {daoData.length} Total DAO{daoData.length === 1 ? '' : 's'}
```

Comment:
V2 had a 'nouns' API that I think we should bring over. Doing this over and over again gets annoying. 

I'll make a ticket